src/components/Modals/SendLLDModal.js

```diff
@@ -4,19 +4,29 @@ import PropTypes from 'prop-types';
 import { useForm } from 'react-hook-form';
 
 // COMPONENTS
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { BN_ZERO, BN } from '@polkadot/util';
 import ModalRoot from './ModalRoot';
 import { TextInput } from '../InputComponents';
 import Button from '../Button/Button';
 
 // STYLES
 import styles from './styles.module.scss';
 import { isValidSubstrateAddress } from '../../utils/bridge';
-import { parseDollars } from '../../utils/walletHelpers';
+import { parseDollars, parseMerits } from '../../utils/walletHelpers';
 import { walletActions } from '../../redux/actions';
+import { walletSelectors } from '../../redux/selectors';
 
 function SendLLDModal({ closeModal }) {
   const dispatch = useDispatch();
+  const balances = useSelector(walletSelectors.selectorBalances);
+  const senderWallet = useSelector(walletSelectors.selectorWalletAddress);
+  const maxUnbond = BN.max(
+    BN_ZERO,
+    (new BN(balances?.liquidAmount?.amount ?? 0))
+      .sub(parseDollars('10')), // leave at least 10 liquid LLD...
```

Comment:
Good catch that minimum should remain, however it shouldnt be 10 and the user should know that he cant send all. I will fix it though